crucible-llvm/src/Lang/Crucible/LLVM/Intrinsics/Common.hs

```diff
@@ -139,53 +139,58 @@ newtype ValTransformer p sym tp tp' =
 
 transformLLVMArgs :: forall m p sym bak args args'.
   (IsSymBackend sym bak, Monad m, HasLLVMAnn sym) =>
+  FunctionName ->
```

Comment:
It would be worth stating in the Haddocks that this `FunctionName` is only used for error-message purposes.